src/components/ec-dropdown-search/ec-dropdown-search.vue

```diff
@@ -300,125 +301,123 @@ export default {
-$ec-dropdown-search-icon-size: 16px !default;
-$ec-dropdown-search-item-height: 40px !default;
-$ec-dropdown-search-item-delimiter-size: 1px !default;
+:root {
+  --ec-dropdown-search-background-color: theme('colors.gray.8');
+  --ec-dropdown-search-background-color-hover: theme('colors.gray.7');
+  --ec-dropdown-search-background-color-selected: theme('colors.key.4');
+  --ec-dropdown-search-background-color-disabled: theme('colors.gray.8');
+  --ec-dropdown-search-color-hover: inherit;
+  --ec-dropdown-search-color-selected: theme('colors.gray.8');
+  --ec-dropdown-search-color-disabled: theme('colors.gray.6');
+  --ec-dropdown-search-border-color: theme('colors.gray.6');
+  --ec-dropdown-search-icon-color: currentColor;
+  --ec-dropddown-search-icon-margin-right: 8px;
+  --ec-dropdown-search-icon-size: 16px;
+  --ec-dropdown-search-item-height: 40px;
+  --ec-dropdown-search-item-delimiter-size: 1px;
+  --ec-dropdown-search-item-horizontal-padding: 8px;
+  --ec-dropdown-search-item-vertical-padding: 16px;
+}
 
 .ec-dropdown-search {
-  $search-icon-margin-right: $ec-dropdown-search-item-horizontal-padding;
-
   &__popover {
-    @include box-shadow-level-1;
-    @include body-text;
+    @apply tw-shadow-level-1;
+    @apply tw-body-text;
 
-    border: 1px solid $ec-dropdown-search-border-color;
-    background-color: $ec-dropdown-search-background-color;
-    color: $ec-dropdown-search-color;
+    border: 1px solid var(--ec-dropdown-search-border-color);
+    background-color: var(--ec-dropdown-search-background-color);
+    color: var(--ec-dropdown-search-color);
   }
 
   &__trigger {
-    // this bit is not configurable in v-tooltip and it's very annoying to deal with
-    // for context:
-    // https://github.com/Akryum/v-tooltip/issues/160
-    // https://github.com/Akryum/v-tooltip/issues/363
-    //
-    // the issue is fixed in v3 alpha, we will remove this hack after updating
+    /* this bit is not configurable in v-tooltip and it's very annoying to deal with
+       for context:
+       https://github.com/Akryum/v-tooltip/issues/160
+       https://github.com/Akryum/v-tooltip/issues/363
+
+       the issue is fixed in v3 alpha, we will remove this hack after updating
+    */
     .trigger {
-      display: block !important;
+      @apply tw-block !important;
     }
   }
 
   &__search-area {
-    position: relative;
-    display: flex;
-    align-items: center;
+    @apply tw-relative;
+    @apply tw-flex tw-items-center;
   }
 
   &__search-area,
   &__cta-area {
-    border-bottom: $ec-dropdown-search-item-delimiter-size solid $ec-dropdown-search-border-color;
+    border-bottom: var(--ec-dropdown-search-item-delimiter-size) solid var(--ec-dropdown-search-border-color);
   }
 
   &__cta-area {
-    @include ec-dropdown-search-item-hover-effect(
-      $background-color: $ec-dropdown-search-background-color-hover,
-      $color: $ec-dropdown-search-color-hover,
-    );
+    /* stylelint-disable */
```

Comment:
We should disable specific rule, e.g. `stylelint-disable selector-max-universal`. We also must re-enable stylelint otherwise it will be disabled to the end of the file. Add `stylelint-enable` after the line you want to ignore.

Reply:
- [ ] Enable stylelint